src/componenets/Dropdown.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+
+class Dropdown extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+        open: false,
+        items: this.props.items,
+        header: this.props.header
+    };
+    this.selectItem = this.selectItem.bind(this);
+    this.toggleDropdown = this.toggleDropdown.bind(this);
+  }   
+  toggleDropdown() {
+    this.setState( (prevState) => ({
+      open: !prevState.open
+    }));
+  }
+
+  selectItem(item) {
+    this.setState({
+      header: item,
+      open: false
+    });
+  }
+
+  renderItems() {
+    if (!this.props.items) {
+      return;
+    }
+
+    return this.props.items.map(item => {
+      return (
+        <li onClick={() => this.selectItem(item)} className="list-item">
+          {item}
+        </li>
+      );
+    });
+  }
+
+  render() {
+    return (
+      <div className="dropdown">
+        <div className="dd-header" onClick={() => this.toggleDropdown()}>
+          <div className="dd-header-title">
+            {this.state.header}
+            {this.state.open ? "^":"v"}
```

Comment:
Instead of using `"^":"v"`, you should export the icon from Figma (let me know if you don't know how) as an SVG and use it here. In order to rotate the icon 180degrees, you can use CSS.